module/zfs/spa.c

```diff
@@ -565,7 +553,7 @@ spa_prop_get(spa_t *spa, nvlist_t **nvp)
 	/*
 	 * Get properties from the spa config.
 	 */
-	spa_prop_get_config(spa, nvp);
+	spa_prop_get_config(spa, &nv);
```

Comment:
I think we should go all the way and update `spa_prop_get_config()` also. I don't see it ever use the `nvp` without `*`.

Reply:
Oh hey! I missed that it was `static`. Easy, done!